src/main/java/Parser.java

```diff
@@ -0,0 +1,66 @@
+import duke.DukeException;
+import java.util.Scanner;
+
+public class Parser {
+    //private TaskList tasks;
+
+    public Parser() {
+        //tasks = new TaskList();
+    }
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+    public static void handleCommands() {
```

Comment:
This function is quite long (>30 lines). Maybe refactor the code inside it to a new function? [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)